pil-analyzer/tests/condenser.rs

```diff
@@ -872,18 +871,15 @@ fn capture_not_supported() {
         })();
 
     "#;
-    let expected = r#"namespace std::prelude;
-    let challenge = 8;
-namespace std::prover;
+    let expected = r#"namespace std::prover;
     let provide_value = 9;
     let eval = -1;
+    col witness x;
     col witness y;
     {
-        let x = std::prelude::challenge(0, 4);
-        let y = std::prover::y;
-        let t = 2;
+        let t = std::prover::x + std::prover::y;
         query |i| {
-            std::prover::provide_value(y, i, std::prover::eval(x) + t);
```

Comment:
What should I do instead? It doesn't work with that tests, expected gives the following result: `let _: fe = std::prover::eval(t);`

Reply:
I don't follow. Are you saying that
1) the original test did not test your changes
2) the original test is still failing even with your changes
?

Reply:
First one, the original test did not test your changes since I think the test should test whether let = x + y; transforms into let t = std::prover::x + std::prover::y;

Reply:
Ah sorry! I thought this was the input, not the expectation :)

Reply:
The expected part of the original test is the same as the test for capture_challenges_and_numbers, which seems it is quite unrelated with the capture_not_supported test? https://github.com/powdr-labs/powdr/blob/718c333ff960d5c7e4f254397cc3702a0636f736/pil-analyzer/tests/condenser.rs#L836-L854